pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -6,6 +6,12 @@ import (
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
 )
 
+// DefaultFieldMaxLength is the standard size for most user-settable string fields in Alerting. Use this for typical string fields, unless you have a special reason not to.
+const DefaultFieldMaxLength = 190
+
+// UIDMaxLength is the standard size for fields that contain UIDs.
+const UIDMaxLength = 40
```

Comment:
THANK YOU  for extracting these as constants.

Reply:
:bow: